lib/asm/arm64/Arm64Assembler.v3

```diff
@@ -53,32 +57,31 @@ component Arm64Regs {
 	def R26 = Arm64Gpr.new("x26", "w26", 26);	def R27 = Arm64Gpr.new("x27", "w27", 27);
 	def R28 = Arm64Gpr.new("x28", "w28", 28);	def R29 = Arm64Gpr.new("x29", "w29", 29);
 	def R30 = Arm64Gpr.new("x30", "w30", 30);	def SP = Arm64Gpr.new("sp", "wsp", 31);
-	def RZR = Arm64Gpr.new("xzr", "wzr", 32);
-
-	def V0 = Arm64Sfr.new("v0", 0);		def V1 = Arm64Sfr.new("v1", 1);
-	def V2 = Arm64Sfr.new("v2", 2);		def V3 = Arm64Sfr.new("v3", 3);
-	def V4 = Arm64Sfr.new("v4", 4);		def V5 = Arm64Sfr.new("v5", 5);
-	def V6 = Arm64Sfr.new("v6", 6);		def V7 = Arm64Sfr.new("v7", 7);
-	def V8 = Arm64Sfr.new("v8", 8);		def V9 = Arm64Sfr.new("v9", 9);
-	def V10 = Arm64Sfr.new("v10", 10);		def V11 = Arm64Sfr.new("v11", 11);
-	def V12 = Arm64Sfr.new("v12", 12);		def V13 = Arm64Sfr.new("v13", 13);
-	def V14 = Arm64Sfr.new("v14", 14);		def V15 = Arm64Sfr.new("v15", 15);
-	def V16 = Arm64Sfr.new("v16", 16);		def V17 = Arm64Sfr.new("v17", 17);
-	def V18 = Arm64Sfr.new("v18", 18);		def V19 = Arm64Sfr.new("v19", 19);
-	def V20 = Arm64Sfr.new("v20", 20);		def V21 = Arm64Sfr.new("v21", 21);
-	def V22 = Arm64Sfr.new("v22", 22);		def V23 = Arm64Sfr.new("v23", 23);
-	def V24 = Arm64Sfr.new("v24", 24);		def V25 = Arm64Sfr.new("v25", 25);
-	def V26 = Arm64Sfr.new("v26", 26);		def V27 = Arm64Sfr.new("v27", 27);
-	def V28 = Arm64Sfr.new("v28", 28);		def V29 = Arm64Sfr.new("v29", 29);
-	def V30 = Arm64Sfr.new("v30", 30);		def V31 = Arm64Sfr.new("v31", 31);
+	def RZR = Arm64Gpr.new("xzr", "wzr", 31);
+
+	def V0 = Arm64Sfr.new("v0", 0);				def V1 = Arm64Sfr.new("v1", 1);
```

Comment:
You should be able to put this on the same line by using , to separate multiple defs.